deploy/infrastructure/dependencies/terraform-commons-dss/variables.tf

```diff
@@ -147,23 +150,3 @@ variable "kubernetes_namespace" {
   }
 }
 
-variable "app_hostname" {
```

Comment:
This whole block with `app_hostname` and `crdb_hostname_suffix` needs to be re-added back, this breaks `terraform plan` command. 

![image](https://user-images.githubusercontent.com/1588744/215828014-b6312f09-549b-424f-b47f-c9acae2bc37c.png)

Reply:
Thank you @hrishiballal  for the feedback, may I kindly ask you to test the latest version of this branch. I just pushed a fix.